platform/understand/externally-vs-platform-deployed.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Virtual Cluster Lifecycle Ownership
+sidebar_label: Virtual Cluster Lifecycle Ownership
+---
+
+# Virtual Cluster Lifecycle Ownership
+
+There are two main types of virtual clusters based on how they're managed: platform-managed and externally managed. These classifications determine how the clusters are created, updated, and integrated with the platform's features. Understanding the differences between these types is important for effective cluster management and utilization of platform capabilities.
+
+## Platform-Managed Virtual Clusters
+
+These are virtual clusters that are deployed and fully managed through our platform. They benefit from the platform's complete set of management features and automated lifecycle management.
+
+:::info Characteristics
+- Identified by `external: false` in `VirtualClusterInstance` CR
+- Deployment methods:
+    - Platform UI
+    - Platform CLI (when authenticated to Platform)
+    - `vcluster platform create vcluster` command
+    - `vcluster create <name> --driver platform` command
+- `external: false` is set automatically when using platform deployment methods
+- Full platform lifecycle management and reconciliation
+- Complete access to platform-specific features and tooling
+:::
+
+Platform-managed clusters provide an integrated experience with centralized management and full platform feature utilization.
+
+## Externally Managed Virtual Clusters
+
+These virtual clusters are initially deployed outside of our platform, using tools like Helm or custom scripts. While they can be imported into the platform for some level of management, they retain their original ownership and have limited platform integration.
+
+:::info Characteristics
+- Deployed using external tools (e.g., ArgoCD, CAPI)
+- Deployed using `vcluster create <name>` command (if driver helm is active)
+- Deployed using `vcluster create <name> --driver helm`
+- Deployed using `helm install vcluster ...`
+- Added to the platform using `vcluster platform add vcluster` command
+- Platform won't manage it (perform reconciliation)
+:::
+
+In this scenario, changes to externally managed virtual clusters should not be performed via the Platform UI. Instead, they should be made using the same tool originally used to deploy the virtual cluster (for example ArgoCD, ClusterAPI, or other tools).
+
+## Understanding the Differences
+
+Virtual clusters are classified based on two key factors: their creation method (via the platform or externally) and their update mechanism (managed by the platform or an external tool). These factors determine whether the platform or an external tool is the lifecycle owner.
+
+:::note
+In short, the key differences are:
+- Creation method: Platform interface vs. external tools
+- Update mechanism: Platform-driven vs. externally controlled
+:::
+
+📝 Current limitations for externally managed clusters:
```

Comment:
These should be removed as this is not a limitation if the cluster has an agent.

Reply:
My understanding is that an externally managed/owned/deployed virtual cluster doesn’t yet support those features regardless if platform agent runs on the host cluster.
Confirmed this on our slack.

Reply:
No, that's wrong, you definitely can use sleep mode as long as you have the agent. We added in vcluster.yaml support for it but it needs an agent aka. you need to connect the host cluster to the platform.

Here are the docs on how to use it: https://vcluster.com/docs/vcluster/configure/vcluster-yaml/external/platform/auto-sleep

Where on slack or with who did you confirm with?

Reply:
It was [this slack thread](https://loft-labs-inc.slack.com/archives/C058PCKE17W/p1728294935542659), I didn't see it was later updated by Kurt with additional answers.

Changing the section to reflect the feedback.